go/saltpack/const.go

```diff
@@ -39,6 +39,22 @@ const EncryptionArmorHeader = "BEGIN KEYBASE ENCRYPTED MESSAGE"
 // armored KB message.
 const EncryptionArmorFooter = "END KEYBASE ENCRYPTED MESSAGE"
 
+// SignedArmorHeader is the header that marks the start of signed
+// armored KB message.
+const SignedArmorHeader = "BEGIN KEYBASE SIGNED MESSAGE"
```

Comment:
BTW, does that look gross?  It seems like too much stuff... hmmm....

`Begin Keybase SaltPack signed message.`

Would that be better?